pkg/raft/raft.go

```diff
@@ -1751,6 +1755,7 @@ func (r *raft) handleAppendEntries(m pb.Message) {
 		To:         m.From,
 		Type:       pb.MsgAppResp,
 		Index:      m.Index,
+		Commit:     r.raftLog.committed,
```

Comment:
Do we need to attach `Commit` to reject resps? I think it's fine/consistent, but just want to see what others think.

One way to think about the commit index flow is that it's independent from log appends. So it doesn't matter if the append is accepted/rejected, we just use its flow as a side-channel.